homeassistant/components/sensor/plex.py

```diff
@@ -116,8 +118,66 @@ def update(self):
         now_playing = []
         for sess in sessions:
             user = sess.usernames[0] if sess.usernames is not None else ""
-            title = sess.title if sess.title is not None else ""
-            year = sess.year if sess.year is not None else ""
-            now_playing.append((user, "{0} ({1})".format(title, year)))
+            playing_title = []
+            if sess.TYPE == 'movie':
+                # example
+                # "The Incredible Hulk (2008)"
+                mov_title = (sess.title
+                             if sess.title is not None
+                             else None)
+                if sess.year is not None and mov_title is not None:
+                    mov_title += " ({0})".format(sess.year)
+
+                playing_title.append(mov_title)
+            elif sess.TYPE == 'episode':
+                # example:
+                # "Supernatural (2005) - S01 · E13 - Route 666"
+                season_title = (sess.grandparentTitle
+                                if sess.grandparentTitle is not None
+                                else None)
+                if sess.show().year is not None and season_title is not None:
+                    season_title += " ({0})".format(sess.show().year)
+
+                season_episode = ("S{0}".format(sess.parentIndex)
+                                  if sess.parentIndex is not None
+                                  else None)
+                if sess.index is not None and season_episode is not None:
+                    season_episode += " · E{0}".format(sess.index)
+
+                episode_title = (sess.title
+                                 if sess.title is not None
+                                 else None)
+
+                playing_title.extend([season_title, season_episode,
+                                      episode_title])
+            elif sess.TYPE == 'track':
+                # example:
+                # "Billy Talent - Afraid of Heights - Afraid of Heights"
+                track_artist = (sess.grandparentTitle
+                                if sess.grandparentTitle is not None
+                                else None)
+
+                track_album = (sess.parentTitle
```

Comment:
There is no reason for this if…else. If it's `None`, you assign `None`. So you can just assign sess.parentTitle ?